openfe/utils/integration_tools.py

```diff
@@ -0,0 +1,139 @@
+"""
+Tools for integration with miscellaneous non-required packages.
+shamelessly borrowed from openpathsampling author: dwhswenson
+"""
+import functools
+from openff.toolkit.utils import toolkits
+from .errors import MissingPackageError, LicenseError
+
+
+def error_if_no_license(name: str, license_name: str, has_license: bool):
+    if not has_license:
+        raise LicenseError(required_by=name, license_name=license_name)
+
+
+def error_if_no_package(name: str, package_name: str, has_package: bool):
+    if not has_package:
+        raise MissingPackageError(required_by=name, package_name=package_name)
+
+"""
+ ===========================================================================
+    SOLUTION 1
+ ===========================================================================
+"""
+# Licenses
+# OpenEye ###################################################
+
+if (not toolkits.OPENEYE_AVAILABLE):
```

Comment:
this is probably going to make the package print a load of OEChem stuff to stderr on import, which is annoying

Reply:
here openff takes care of it and removes the logging for the testing.

in Solution 2 I would need to add that.